src/frontends/tensorflow_common/src/op/matrix_inverse.cpp

```diff
@@ -0,0 +1,34 @@
+// Contributed by Hongbo Wei
+// https://github.com/hongbo-wei
+// Good first issue: https://github.com/openvinotoolkit/openvino/issues/22957
+
+#include "openvino/op/inverse.hpp"
+
+#include "common_op_table.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_matrix_inverse_op(const NodeContext& node) {
+    auto op_type = node.get_op_type();
```

Comment:
please use helper for check `default_op_checks`. See other PRs and other operations

Reply:
I saw Jane used this in `https://github.com/openvinotoolkit/openvino/issues/22964`, but I'm not sure what activation function to use here.